### PR TITLE
Integrate automatic population creation processes into the existing frameworks

### PR DESCRIPTION
**Pull request by Sticksman**
This is a first pass at integrating @KelWill's work into the existing dags.

### CODE REVIEW

File: dags/experiment_population_creation.py [hunk 1]
```diff
@@ -142,12 +143,34 @@ def get_manually_mapped_tables(conn_id, ts, **kwargs):
     return d
 
 
+def generate_automatic_population(conn_id, ts, **kwargs):
```
Comment (Sticksman) [inline dags/experiment_population_creation.py:146]: This was moved from the automatic population creation module

File: dags/experiment_population_creation.py [hunk 2]
```diff
@@ -163,6 +186,21 @@ def write_mappings(conn_id, ts, **kwargs):
         }
         pg_hook.run(query)
 
+    for experiment_id, table_name in automatic_tables_by_id.items():
```
Comment (Sticksman) [inline dags/experiment_population_creation.py:189]: Moved from the automatic module to here

File: dags/experiment_population_creation.py [hunk 3]
```diff
@@ -142,12 +143,34 @@ def get_manually_mapped_tables(conn_id, ts, **kwargs):
...
+def generate_automatic_population(conn_id, ts, **kwargs):
+    print("creating automatic populations!")
+    lumberjack_experiments = automatic_population_util.get_lumberjack_experiments(
+        conn_id)
+    print("lumberjack_experiments", lumberjack_experiments)
+
+    experiment_names_by_id = []
+
+    for experiment_name, _ in lumberjack_experiments:
+        experiment_id = experiment_name[:36]
```
Comment (KelWill) [inline dags/experiment_population_creation.py:155]: now that we've got `get_experiment_id`, we could use that instead

**Review by KelWill — APPROVED** [review]
This looks awesome!! Thanks :)

**Pull request merged**